Summary 2 on ramachandran2017searching

liaojh1998 opened the pull request:
To be completed by tonight.

liaojh1998 commented:
Draft completed.

kraja99 commented on _summaries/ramachandran2017searching_2.md:
```diff
@@ -1,19 +1,63 @@
+            <img src="https://d3i71xaburhd42.cloudfront.net/c8c4ab59ac29973a00df4e5c8df3773a3c59995a/2-Figure1-1.png" width="30%" height="30%">
+        </p>
+
+        Note that the inputs of the unary functions are restricted to the preactivation $$x$$ and the binary function outputs. This way, the activation function can always take a scalar value as input and output a single scalar value, like ReLU.
```
Maybe give a few examples of the unary or binary functions in the search space.

liaojh1998 replied:
Added all examples for reference.

kraja99 commented on _summaries/ramachandran2017searching_2.md:
```diff
@@ -1,19 +1,63 @@
+
+    * _Search Evaluations_
+        
+        The searches are conducted by training ResNet-20 ([He et al., 2016a](https://ieeexplore.ieee.org/document/7780459)) on the CIFAR-10 ([Krizhevsky & Hinton, 2009](https://www.cs.toronto.edu/~kriz/learning-features-2009-TR.pdf)) by substituting each activation function instead of ReLU. Every network is trained for 10K steps.
```
Consider mentioning the use of "child networks" more explicitly, as well as the use of distributed training to parallelize computation.

liaojh1998 replied:
Done!

kraja99 reviewed (approved):
Overall good summary on the paper!

liaojh1998 commented:
Changes:
* Added all examples of unary and binary functions.
* Made explicit the use of child networks, the fact that they took time to train, so that the authors added a distributed training scheme for search.
* Added comments on differences between ReLU and Swish, which models can take advantage of.

Let me know if there are any other issues. Thank you so much @kraja99 for the suggestions!
